ui/package.json

```diff
@@ -8,44 +8,46 @@
   },
   "author": "Codeloop",
   "license": "ISC",
+  "private": true,
   "devDependencies": {
-    "babel-core": "6.26.x",
-    "babel-loader": "7.1.x",
-    "babel-plugin-transform-react-jsx": "6.24.x",
-    "babel-preset-env": "1.7.x",
-    "babel-preset-stage-2": "6.24.x",
-    "copy-webpack-plugin": "5.0.x",
-    "cross-env": "5.2.x",
-    "css-loader": "3.0.x",
     "eslint": "5.16.x",
     "eslint-config-airbnb": "17.1.x",
     "eslint-loader": "2.2.x",
     "eslint-plugin-import": "2.18.x",
     "eslint-plugin-jsx-a11y": "6.2.x",
-    "eslint-plugin-react": "7.14.x",
-    "file-loader": "4.0.x",
-    "html-webpack-plugin": "3.2.x",
-    "mini-css-extract-plugin": "0.8.x",
-    "node-sass": "4.12.x",
-    "postcss-loader": "3.0.x",
-    "@primer/css": "^12.4.x",
-    "sass-loader": "7.1.x",
-    "style-loader": "0.23.x",
-    "stylelint": "10.1.x",
-    "stylelint-config-primer": "7.0.x",
-    "stylelint-webpack-plugin": "0.10.x",
-    "webpack": "4.35.x",
-    "webpack-cli": "3.3.x",
-    "webpack-dev-server": "3.7.x"
+    "eslint-plugin-react": "7.14.x"
   },
   "dependencies": {
-    "@hyperapp/logger": "0.5.x",
-    "@hyperapp/router": "0.7.x",
-    "autoprefixer": "9.6.x",
-    "bootstrap": "4.3.x",
-    "hyperapp": "1.2.x",
-    "jquery": "3.4.x",
-    "popper.js": "1.15.x",
-    "whatwg-fetch": "3.0.x"
```

Comment:
- [ ] `popper.js` was needed for bootstrap
- [ ] `jquery` was needed for bootstrap
- [ ] `autoprefixer` do we need it?
- [ ] `whatwg-fetch` window.fetch polyfill - do we need it?
- [ ] `mini-css-extract-plugin` does react-scripts minify css?
- [ ] `@primer/css` - The CSS design system that powers GitHub
- [ ] `stylelint`, `stylelint-config-primer`

Reply:
- `popper.js` is still as a dependency
```
$ yarn list | grep popper
├─ @react-bootstrap/react-popper@1.2.1
│  ├─ popper.js@^1.14.4
├─ popper.js@1.15.0
│  ├─ @react-bootstrap/react-popper@1.2.1
│  ├─ popper.js@^1.14.7
│  ├─ react-popper@^1.3.2
├─ react-popper@1.3.3
│  ├─ popper.js@^1.14.4
```
- `jquery` is not listed. I guess it's bundled in `react-bootstrap` or `bootstrap`? Or maybe bootstrap is rewritten without jquery?

- `autoprefixer` is still as a dependency
```
$ yarn list | grep autoprefixer
├─ autoprefixer@9.5.1
│  ├─ autoprefixer@^9.4.9
```

- `whatwg-fetch` I don't know. Will look at it later.

- `mini-css-extract-plugin` I'm pretty sure they minify css in production build.
https://github.com/facebook/create-react-app/issues/1934

- `@primer/css` What is that?

- `stylelint` I missed this. I will add it.

Reply:
- About `@primer/css`
https://github.com/primer/css

Reply:
It looks like primer is another css library. We use bootstrap, so I don't see a use case for primer.

Reply:
* `whatwg-fetch` "Do we need it?" I don't know. But if we want it, we can use [react-app-polyfill](https://www.npmjs.com/package/react-app-polyfill).

Reply:
* `jquery` For only css from bootstrap jquery is not needed. https://stackoverflow.com/a/14608772

Reply:
OK. Primer is needed for stylelint.

Reply:
- `stylelint` I added it and created scripts in package.json (`yarn eslint`, `yarn stylelint`).

Reply:
` react-app-polyfill` is a good idea